src/mspdb.cpp

```diff
@@ -239,6 +327,8 @@ void tryLoadMsPdb140(bool throughPath)
 	{
 		if(throughPath)
 			modMsPdb = LoadLibraryA(mspdb140_dll);
+		if(!modMsPdb && !throughPath)
+			tryLoadMsPdbVSWhere(mspdb140_dll);

```

Comment:
Would it also work for you to move this after the checks through COM and the registry? This avoids changing existing uses and the other alternatives are probably faster than creating a process.

Reply:
I believe it does work!